<launch>
	<env name="TESSDATA_PREFIX" value="~/git/care-o-bot/cob_object_perception_data/cob_tesseract_data" />

	<!--node name="image_proc" pkg="image_proc" type="image_proc"
		ns="prosilica" /-->

	<!--node name="image_view_color" pkg="image_view" type="image_view">
		<remap from="image" to="prosilica/image_rect_color"/>
	</node-->

	<node name="image_view_detect" pkg="image_view" type="image_view">
		<remap from="image" to="/read_text/text_detect"/>
	</node>

		<!-- args="$(find read_text)/fonts/correlation.txt 
		$(find read_text)/dictionary/full-dictionary"> -->
	<node name="cob_read_text" pkg="cob_read_text" type="cob_read_text" output="screen"
		args="/home/rmb-rh/git/care-o-bot/cob_object_perception_data/cob_read_text_data/fonts/correlation.txt 
		/home/rmb-rh/git/care-o-bot/cob_object_perception_data/cob_read_text_data/dictionary/full-dictionary">
		<remap from="text_detect" to="/read_text/text_detect"/>
		<remap from="image_color" to="/camera/rgb/image_color"/>
	</node>
</launch>
